src/content/glossary/index.md

```diff
@@ -460,6 +553,10 @@ A [transaction](#transaction) sent from a [contract account](#contract-account)
 
 <Divider />
 
+### issuance
+
+The minting and granting of new cryptocurrency to a miner who has found a new block.
+
```

Comment:
This is not fully accurate since the launch of the Beacon Chain... Issuance on the Beacon Chain is not just from block proposals, but also for attestations and whistleblowing. 
https://twitter.com/trent_vanepps/status/1509538632016801792

Reply:
yep, fair. Updated.